tests/conftest.py

```diff
@@ -56,20 +59,32 @@ def agent_index_dir(agent_home_dir: Path) -> Path:
 
 
 @pytest.fixture
-def agent_stub_answer() -> Answer:
-    return Answer(question="What is is a self-explanatory model?")
+def stub_data_dir() -> Path:
+    return Path(__file__).parent / "stub_data"
 
 
 @pytest.fixture
-def stub_data_dir() -> Path:
-    return Path(__file__).parent / "stub_data"
+def agent_test_settings(agent_index_dir: Path, stub_data_dir: Path) -> Settings:
+    settings = Settings()
+    settings.agent.paper_directory = stub_data_dir
+    settings.agent.index_directory = agent_index_dir
+    settings.agent.search_count = 2
+    settings.embedding = "sparse"
```

Comment:
Can we configure this in the instantiation of `Settings(embedding="sparse")`?

Reply:
Not opposed - but why?